main.go

```diff
@@ -45,4 +55,15 @@ func main() {
 	close(swapIntentOperationBroadcastChan)
 	close(solverOperationSubmitChan)
 	close(governanceSignatureChan)
+
+	// Questions:
+	// entities:
+	//    backend - is this the bundler that gets the intent, distribute to solvers, collect solutio and create the metaTransactions? you can it backend and not bundler as there can be other configurations?
```

Comment:
The backend is the entity orchestrating the whole process.
- It gets intents
- Distribute to solvers
- Collect solutions
- Gets governance signature
In this example it's also acting as bundler, but it doesn't have to be. After collecting intent/solutions/governanceSignature, the backend could send all of this to another entity, that would call Atlas.metacall, that would make that entity the bundler.
We call the bundler the entity that calls Atlas.metacall and therefore fronts gas cost.

Reply:
ok